backend/tests/utils/mock_redis.py

```diff
@@ -0,0 +1,85 @@
+import json
+from unittest import mock
+
+
+# TODO - should we use FakeRedis instead like filenameprocessor?
+# TODO - or should we just mock individual hget / hkeys calls?
+class MockRedisClient:
```

Comment:
I don't really like this approach - I think we should mock the exact `hget` / `hkeys` calls we expect in each individual test instead, like we've done for the supplier permissions tests.